.swiftlint.yml

```diff
@@ -3,6 +3,10 @@ disabled_rules: # rule identifiers turned on by default to exclude from running
   # TODOs and FIXMEs should be resolved.
   - todo
   
+  # swiftlint:disable commands should be re-enabled before the end of the file
+  # Needed for auto generated files from templates
+  - blanket_disable_command
```

Comment:
This issue blocks the project building for some Xcode versions. More on [Slack](https://motooniosteam.slack.com/archives/C055XKKBQH4/p1684660531976099)